### ISSUE TITLE
Filed to install with python 3.7

### ISSUE THREAD

**Issue by johngull** [TylerGubala/blenderpy#26]
**Describe the bug**
Installing blenderpy on Ubuntu with the python 3.7.
It show next error:
```
    ERROR: Command errored out with exit status 1:
     command: /home/vitaly/miniconda3/envs/blenderpy-test/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-1i9rexzd/bpy/setup.py'"'"'; __file__='"'"'/tmp/pip-install-1i9rexzd/bpy/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: /tmp/pip-install-1i9rexzd/bpy/
    Complete output (16 lines):
    Traceback (most recent call last):
      File "/home/vitaly/miniconda3/envs/blenderpy-test/lib/python3.7/tokenize.py", line 397, in find_cookie
        codec = lookup(encoding)
    LookupError: unknown encoding: future_fstrings
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/vitaly/miniconda3/envs/blenderpy-test/lib/python3.7/tokenize.py", line 449, in open
        encoding, lines = detect_encoding(buffer.readline)
      File "/home/vitaly/miniconda3/envs/blenderpy-test/lib/python3.7/tokenize.py", line 436, in detect_encoding
        encoding = find_cookie(second)
      File "/home/vitaly/miniconda3/envs/blenderpy-test/lib/python3.7/tokenize.py", line 405, in find_cookie
        raise SyntaxError(msg)
    SyntaxError: unknown encoding for '/tmp/pip-install-1i9rexzd/bpy/setup.py': future_fstrings
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
I tried with the python 3.5 and installation worked fine.

**Desktop (please complete the following information):**
 - Ubuntu

**Comment by no-materials** [comment]
I have the exact same issue.

**Comment by TylerGubala** [comment]
Sorry, Ubuntu isn' supported yet. Same as #1.

MacOS support is coming, then Ubuntu.

**Comment by TylerGubala** [comment]
Closed as same as #1
